R/export-dataset.R

```diff
@@ -46,7 +46,8 @@ variablesFilter <- function (dataset, map=FALSE) {
                 .Names=ids(allvars))))
         } else {
             ## Old
-            return(list(`function`="identify", args=list(list(id=I(urls(allvars))))))
+            return(list(`function`="select", args=list(list(map=structure(lapply(urls(allvars), function (x) list(variable=x)),
+                .Names=urls(allvars))))))
```

Comment:
This is updating the "Old" statement, I don't know where the "new" scenario is used, not supported by the API as it sends a map straight instead of a function, I think should be removed unless it has some case that needed to be supported?